matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -1124,10 +1124,10 @@ var PhenoTips = (function (PhenoTips) {
         return table;
     },
 
-    _generateRecipientCell : function(info, type)
+    _generateRecipientCell : function(info, type, isFrom)
```

Comment:
If `isFrom`, then this is not really a `recipient` cell. The name of the function is inaccurate and confusing. Can we call it something more generic such as `_generateUserInfoCell`?